db/migrate/20160305220019_create_blocks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBlocks < ActiveRecord::Migration
+  def change
+    create_table "blocks", primary_key: "blockid", force: :cascade do |t|
+      t.integer "blocked_userid",  limit: 2, default: 0, null: false
+      t.integer "blocking_userid", limit: 2, default: 0, null: false
+    end
+
+    add_index "blocks", ["blocked_userid", "blocking_userid"], name: "pair", unique: true
+    add_index "blocks", ["blocked_userid"], name: "blocked"
+    add_index "blocks", ["blocking_userid"], name: "blocking"
+  end
+end
```

Comment:
Hmm, we've gotten ourselves into a weird corner here because the database migration for blocking happened in the PHP version *after* this schema was set. So it already exists over there (and in the production DB we'll eventually be using).

This migration should definitely use the same field names as the PHP migration: https://github.com/grinnellplans/grinnellplans-php/blob/master/db/migrations/7_add_blocks_table.php

And I'm thinking I might wrap this one to only have an effect in `development`. That's a pretty shady thing to do in a migration, but it's the simplest and least-disruptive solution I can think of.